code-examples/php/usage-examples/updateOne.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+require 'vendor/autoload.php';
+
+// Replace the uri string with your MongoDB deployment's connection string
+$uri = "mongodb://localhost:27017";
+
+$client = new MongoDB\Client($uri);
+
+// Get the "movies" collection in the "sample_mflix" database
+$collection = $client->sample_mflix->movies;
+
+try {
+    /* Update a document that has the title "Random Harvest" to have a
+    plot field with the specified value */
```

Comment:
Multi-line comments would typically be formatted like so:

```suggestion
    /* Update a document that has the title "Random Harvest" to have a
     * plot field with the specified value */
```

Also confirm that the wrap point is what you intend. 80-char is probably ideal for this.